packages/owl-bot/package.json

```diff
@@ -60,6 +61,7 @@
   "dependencies": {
     "@google-cloud/cloudbuild": "^2.0.6",
     "@octokit/rest": "^18.5.3",
+    "adm-zip": "^0.5.5",
```

Comment:
Wondering if we could get away with using Node's built in Zlib implementation, vs., pulling in dep?

https://nodejs.org/api/zlib.html#zlib_class_zlib_unzip

Reply:
I read zlib's documentation, and I don't see a way to unpack a zip archive using zlib.  Do you see a way?  All zlib's functions appear to work on single file streams, not archives.